src/v2/Components/Pagination.tsx

```diff
@@ -35,11 +35,15 @@ export class Pagination extends React.Component<Props> {
     return (
       <ScrollIntoView selector={this.props.scrollTo}>
         <Media at="xs">
+          {/* FIXME: */}
+          {/* @ts-ignore */}
           <SmallPagination {...this.props} />
         </Media>
         <Media greaterThan="xs">
           <Box>
             <Separator mb={3} pr={2} />
+            {/* FIXME: */}
+            {/* @ts-ignore */}
```

Comment:
Ah, I see @eessex beat me to this @jonallured — this should be fixable. I'll still maintain we don't need to pass in the cursors, but there are other ways to fix this too